locales/it/index.yml

```diff
@@ -2044,6 +2049,12 @@ bonus:
           active: Attivo
           closed: Concluso
           inactive: In arrivo
+      superCashback:
+        rankLabel: "{{position}}º utente in classifica"
+        rankTitle: "Classifica Super Cashback"
+        howItWorks:
+          title: "Come funziona?"
+          body: "I **100mila cittadini** con il più alto numero di transazioni **ricevono il Super Cashback, un premio pari a €1.500.**\n\n**La classifica cambia ogni giorno.** Quella definitiva sarà visualizzata il {{endDate}}."
```

Comment:
In this case we should use `bpdPeriod.minPosition` instead of the hardcoded text "100mila" (the min ranking could be different for different periods).
We should also use `bpdPeriod.superCashbackAmount`instead of the hardcoded text "€1.500" (the max cashback amount could be different for different periods)